test/onnx/test_onnx_opset.py

```diff
@@ -107,5 +107,42 @@ def test_maxpool(self):
         x = torch.randn(20, 16, 50)
         check_onnx_opsets_operator(module, x, ops, opset_versions=[10])
 
+    def test_slice(self):
+        class MyModule(Module):
+            def forward(self, x):
+                return x[0:1]
+
+        ops_9 = [{"op_name" : "Slice",
+                  "attributes" :
+                  [{"name": "axes", "ints": [0], "type": 7},
```

Comment:
Nit: use ENUM instead of number for "type"

Reply:
will update all of the types for enum in another PR